src/Hazelcast.Net.Tests/Cloud/CloudTests.cs

```diff
@@ -95,15 +94,7 @@ public async Task SampleClient(bool previewOptions)
             options.Metrics.Enabled = true;
 
             // enable reconnection
-            if (previewOptions)
-            {
-                options.Preview.EnableNewReconnectOptions = true;
-                options.Preview.EnableNewRetryOptions = true;
-            }
-            else
-            {
-                options.Networking.ReconnectMode = ReconnectMode.ReconnectAsync;
-            }
+            options.Networking.ReconnectMode = ReconnectMode.ReconnectAsync;
```

Comment:
I think it should be `ReconnectSync` since test doesn't expect to face an exception.

Reply:
ok